service/matching/db_task_manager.go

```diff
@@ -92,26 +92,39 @@ type (
 		taskReader                dbTaskReader
 		taskWriter                dbTaskWriter
 		maxDeletedTaskIDInclusive int64 // in mem only
+		group                     goro.Group
+		dbTaskDeletionInterval    time.Duration
+		dbTaskUpdateAckInterval   time.Duration
+		dbTaskUpdateQueueInterval time.Duration
 	}
 )
 
 func newDBTaskManager(
-	taskQueueKey persistence.TaskQueueKey,
+	registry namespace.Registry,
+	taskQueue *taskQueueID,
 	taskQueueKind enumspb.TaskQueueKind,
 	taskIDRangeSize int64,
 	dispatchTaskFn func(context.Context, *internalTask) error,
 	store persistence.TaskManager,
-	namespaceRegistry namespace.Registry,
 	logger log.Logger,
+	dbTaskDeletionInterval time.Duration,
+	dbTaskUpdateAckInterval time.Duration,
+	dbTaskUpdateQueueInterval time.Duration,
```

Comment:
you should pass in the dynamic config, not the concrete value

Reply:
What's difference? Given those values are only read once during ticker initialization, even we pass in dynamic config, the new values won't be read anyway.